content/en/getting_started/synthetics/api_test.md

```diff
@@ -84,31 +92,31 @@ An alert is triggered if your test fails for 3 minutes from any 2 of 13 location
 
 ### Notify your team
 
-Add an alert name to the **Monitor Name** field and write a message for the alert. You can use [integrations][6] such as Slack, PagerDuty, and webhooks to route your alert to specific services and teams.
+Add an alert name to the **Monitor Name** field and write a message for the alert. You can use [integrations][7] such as Slack, PagerDuty, and webhooks to route your alert to specific services and teams.
 
 You can set time for your alert notification to re-notify if the alert has not been resolved and define the priority of the alert, ranging from **P5 (Info)** to **P1 (Critical)**.
 
 When you're ready to run your test, click **Save Test**. 
 
 ## Create a multistep API test
 
-[Multistep API tests][7] run HTTP requests in sequence, allowing you to configure [HTTP tests][2] on complex backend systems. 
+[Multistep API tests][8] run HTTP requests in sequence, allowing you to configure [HTTP tests][2] on complex backend systems. 
 
 ### Configure the test
 
-When you create a [Multistep API test][7], you can define a sequence of HTTP requests and create variables from the response data to re-inject their values in subsequent steps. 
+When you create a [Multistep API test][8], you can define a sequence of HTTP requests and create variables from the response data to re-inject their values in subsequent steps. 
 
 1. In the Datadog application, hover over **[UX Monitoring][3]** in the left hand menu and select **[Synthetic Tests][3]**.
 2. In the top right corner, click the **New Test** button.
-3. Select **[Multistep API test][8]**.
-4. Name, tag and select locations for your [Multistep API test][8] like you would with a single [API test][1].
+3. Select **[Multistep API test][9]**.
+4. Name, tag and select locations for your [Multistep API test][9] like you would with a single [API test][1].
 5. In **Define steps**, click **Create Your First Step**. 
 
     - Add a name to your step.
     - Specify the URL you want to query. 
     - Select **Advanced Options** to add custom request headers, authentication credentials, body content, or cookies.
     - Click **Test URL**. 
-    - Optionally, add new assertions, define execution parameters, and extract variables from the response content.
+    - Optionally, add new assertions, define execution parameters, and extract variables from the response content. When you extract a variable from the response content, remember to re-inject the variable in subsequent steps.
```

Comment:
Hm, this wording makes it sound like it's mandatory. How about we give an example of variable extraction and re-injection instead here? That's the most important bit for multistep API tests so it could be worth adding a screenshot or two. I'm thinking maybe we could go more quickly over what's common to single and multistep API tests and really focus on variable extraction/re-injection in this section?

Reply:
For sure! Do you have a screenshot or video snippet handy for variable extraction/re-injection?

Reply:
We have ss2-exa-git on the demo account, but I'm not a big fan of this one since it's using regex for parsing. I'd be in favor of using an endpoint that responds with a JSON payload. So we could probably use https://reqres.in/ to create a flow. This might require more work so it might be worth creating a dedicated card about that one and sync about it over Slack/Zoom.

Reply:
Cool, added the above to DOCS-2252, which we can sync about in our next meeting.